_commands/tools/dc.md

```diff
@@ -2,22 +2,62 @@
 ---
 dc
 ----
-TODO: Add documentation for this command by submitting a pull request.
 <!-- one line explanation would go here -->
+dc is a desk calculator that supports unlimited-precision arithmetic and postfix notation.
 
 <!-- minimal example -->
 ~~~ bash
+dc
```

Comment:
when showing example bash commands, it's a good idea to use `$ ` just to indicate which part is the command and which is the output

i.e.
```
$ dc
2
3
...
```